packages/utils/interactive-hooks/src/Pressability/Pressability.types.ts

```diff
@@ -64,7 +34,9 @@ export type PressabilityConfig = Readonly<{
    * Called after the element is focused.
    */
   onFocus?: (event: FocusEvent) => any;
```

Comment:
Note: I think it's an open action item that we need to add onFocus/onBlur to RN-macOS pressable. I'm not sure how that affects FURN downstream.

Reply:
Likely the events never trigger but we should make sure that is on the 0.64 list.